src/app/feature-flag/notification-footer/feature-footer.component.spec.ts

```diff
@@ -107,6 +107,21 @@ describe('FeatureFooterComponent', () => {
     hostComponent.featurePageConfig = config;
   });
 
+  it('should display feature descriptions', async(() => {
+    hostComponent.descriptionPerLevel();
+    expect(hostComponent.experimentalFeatureText).toEqual('is 1 experimental feature');
+    expect(hostComponent.internalFeatureText).toEqual('is 1 internal feature');
+    expect(hostComponent.betaFeatureText).toEqual('is 1 beta feature');
+    config.featuresPerLevel.beta.push({'attributes':
+      {'description': 'Planner menu',
+        'enabled': true,
+        'enablement-level': 'beta',
+        'user-enabled': true,
+        'name': 'Planner featureC'},
+      'id': 'Planner.featureC'});
+    expect(hostComponent.betaFeatureText).toEqual('are 2 beta features');
```

Comment:
The test is failing here, I'm seeing: 
✖ should display feature descriptions
      PhantomJS 2.1.1 (Linux 0.0.0)
    Expected 'is 1 beta feature' to equal 'are 2 beta features'.

Reply:
I think you're missing a call to `hostComponent.descriptionLevel()` after you push the new feature (would be line 122), otherwise nothing has changed. I tried it locally and it passes, can you confirm?